apps/communications/contacts/views/details/js/details_ui.js

```diff
@@ -0,0 +1,463 @@
+'use strict';
```

Comment:
You can move this to https://github.com/mozilla-b2g/gaia/pull/30750/files#diff-53dea38d3841fcc8551cbbb553fb4171R10 due to the code is executed within the function.